src/wallet/rpcwallet.cpp

```diff
@@ -910,7 +935,13 @@ UniValue sendmany(const JSONRPCRequest& request)
             "      \"address\"          (string) Subtract fee from this address\n"
             "      ,...\n"
             "    ]\n"
-            "\nResult:\n"
+            "6. replaceable            (boolean, optional) Allow this transaction to be replaced by a transaction with higher fees via BIP 125\n"
```

Comment:
I saw the above comment about named arguments. IMO from the client side feels better as an option.